lib/ContractInteract/TokenHolder.js

```diff
@@ -139,6 +139,66 @@ class TokenHolder {
     return executeRuleCallPrefix;
   }
 
+  /**
+   * It is used to get call prefix of executeRedeem method in TokenHolder contract.
+   *
+   * @returns {string} Encoded signature of executeRedeem method.
+   */
+  getTokenHolderExecuteRedemptionCallPrefix() {
+    const executeRedeemHash = this.auxiliaryWeb3.utils.soliditySha3(
+      'executeRedemption(address,bytes,uint256,bytes32,bytes32,uint8)'
+    );
+    const executeRedeemCallPrefix = executeRedeemHash.substring(0, 10);
```

Comment:
this probably deserves a comment that "0x" + first 4 bytes (8 char) are the selected here

Reply:
please also change it in tokenholder.js `executeRule`